app/services/reporting/total_user_count_report.rb

```diff
@@ -45,6 +46,12 @@ def new_user_count
       end
     end
 
+    def total_fully_registered
+      Reports::BaseReport.transaction_with_timeout do
+        RegistrationLog.where.not(registered_at: nil).count
```

Comment:
let's include `report_date` here so it can be [backdated](https://handbook.login.gov/articles/queries.html#fully-registered-users) like the other queries 
```suggestion
        RegistrationLog.where('registered_at > ?', end_date).where.not(registered_at: nil).count
```